### PR TITLE
[BULK UPDATE] DocuTune - Fix build validation issues: docs-link-absolute (part 1)

### PR DESCRIPTION
**Pull request by alexbuckgit**
[BULK UPDATE] DocuTune - Fix build validation issues: docs-link-absolute

**Global effort to fix build validation errors**

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR was generated using DocuTune. It includes only build validation fixes and does not change other content.

DocuTune PRs improve quality of technical content by finding and automatically correcting a broad set of issues only where the correction is suitable. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

InitiativeId: docutune-build-validation-issues-2022-05
PointOfContact: @MonicaRush
CorrelationId: 0df9c2d6-63f8-4d1e-8b57-b33611a06514

### CODE REVIEW

File: docs/access/desktop-database-reference/transform-statement-microsoft-access-sql.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ The TRANSFORM statement has these parts:
...
+TRANSFORM is optional but when included is the first statement in an SQL string. It precedes a SELECT statement that specifies the fields used as row headings and a [GROUP BY](/office/vba/access/Concepts/Structured-Query-Language/group-by-clause-microsoft-access-sql) clause that specifies row grouping. Optionally, you can include other clauses, such as [WHERE](https://docs.microsoft.com/office/vba/access/Concepts/Structured-Query-Language/where-clause-microsoft-access-sql), that specify additional selection or sorting criteria. You can also use subqueries as predicates — specifically, those in the WHERE clause — in a crosstab query.
```
Comment (MeeraDi) [inline docs/access/desktop-database-reference/transform-statement-microsoft-access-sql.md:63]: The WHERE link was skipped

File: docs/access/desktop-database-reference/union-operation-microsoft-access-sql.md [hunk 2]
```diff
@@ -66,7 +66,7 @@ Use aliases only in the first SELECT statement because they are ignored in any o
 
 > [!NOTE]
 > - You can use a [GROUP BY](https://docs.microsoft.com/office/vba/access/Concepts/Structured-Query-Language/group-by-clause-microsoft-access-sql) or [HAVING](https://docs.microsoft.com/office/vba/access/concepts/structured-query-language/having-clause-microsoft-access-sql) clause in each *query* argument to group the returned data.
```
Comment (MeeraDi) [inline docs/access/desktop-database-reference/union-operation-microsoft-access-sql.md:68]: The GROUP BY nad HAVING links were skipped

File: docs/access/desktop-database-reference/sql-expressions.md [hunk 3]
```diff
@@ -15,10 +15,10 @@ ms.localizationpriority: high
...
+The Microsoft Access database engine uses the Microsoft Visual Basic for Applications (VBA) expression service to perform simple arithmetic and function evaluation. All of the operators used in Microsoft Access database engine SQL expressions (except **[Between](https://docs.microsoft.com/office/vba/access/concepts/miscellaneous/between-and-operator)**, **[In](/office/vba/access/concepts/miscellaneous/in-operator-microsoft-access-sql)**, and **[Like](https://docs.microsoft.com/office/vba/access/Concepts/Structured-Query-Language/like-operator-microsoft-access-sql)**) are defined by the VBA expression service.
```
[the remark this answers is not in the document]
Comment (alexbuckgit) [reply docs/access/desktop-database-reference/sql-expressions.md:18]: Corrected

File: docs/access/desktop-database-reference/sql-subqueries-microsoft-access-sql.md [hunk 4]
```diff
@@ -64,7 +64,7 @@ A subquery has these parts:
 
 ## Remarks
 
-You can use a subquery instead of an expression in the field list of a SELECT statement or in a [WHERE](https://docs.microsoft.com/office/vba/access/Concepts/Structured-Query-Language/where-clause-microsoft-access-sql) or [HAVING](https://docs.microsoft.com/office/vba/access/concepts/structured-query-language/having-clause-microsoft-access-sql) clause. In a subquery, you use a SELECT statement to provide a set of one or more specific values to evaluate in the WHERE or HAVING clause expression.
+You can use a subquery instead of an expression in the field list of a SELECT statement or in a [WHERE](https://docs.microsoft.com/office/vba/access/Concepts/Structured-Query-Language/where-clause-microsoft-access-sql) or [HAVING](/office/vba/access/concepts/structured-query-language/having-clause-microsoft-access-sql) clause. In a subquery, you use a SELECT statement to provide a set of one or more specific values to evaluate in the WHERE or HAVING clause expression.
```
[the remark this answers is not in the document]
Comment (alexbuckgit) [reply docs/access/desktop-database-reference/sql-subqueries-microsoft-access-sql.md:67]: Corrected

File: docs/access/desktop-database-reference/transform-statement-microsoft-access-sql.md (hunk 1 above)
Comment (alexbuckgit) [reply docs/access/desktop-database-reference/transform-statement-microsoft-access-sql.md:63]: Corrected

File: docs/access/desktop-database-reference/union-operation-microsoft-access-sql.md (hunk 2 above)
Comment (alexbuckgit) [reply docs/access/desktop-database-reference/union-operation-microsoft-access-sql.md]: Corrected

**Comment by alexbuckgit** [comment]
#docutune-ready

**Comment by alexbuckgit** [comment]
@AlexJerabek @dstrome

Please merge the changes in this PR. For more information, see the description at the top of this PR.

Any pre-existing build validation issues or insufficient Acrolinx scores should be addressed in a separate PR.

#docutune-request-approval

**Pull request merged**